utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
+    init()
+except ImportError:  # fallback so that the imported classes always exist
+    class ColorFallback():
+        __getattr__ = lambda self, name: ''
+    Fore = Back = Style = ColorFallback()
+
+def color_diff(diff):
+    '''Colorize lines in the diff generator object'''
+
+    for line in diff:
+        if line.startswith('+'):
+            yield Fore.GREEN + line + Fore.RESET
+        elif line.startswith('-'):
+            yield Fore.RED + line + Fore.RESET
+        else:
+            yield line
+
+class Minion():
+
+    default_config = '/opt/{name}/minion'
```

Comment:
Default config is mutable class field. this is ok for this simple file but in general it's much better to declare it as readonly property or rename as DEFAULT_CONFIG that means 'please dont write to this var' :)